packages/styled-components/src/models/StyleSheetManager.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import React, { useContext, useMemo, useState, type Node, type Context } from 'react';
 import PropTypes from 'prop-types';
+import shallowequal from 'shallowequal';
```

Comment:
Remains to be seen whether this is correctly minified away I suppose

Reply:
I checked and it gets DC'd properly due to the NODE_ENV wrapper 👍